common/persistence/cassandra/cassandraMetadataPersistenceV2.go

```diff
@@ -435,7 +435,8 @@ func (m *cassandraMetadataPersistenceV2) GetDomain(request *p.GetDomainRequest)
 
 	var responseFailoverEndTime *int64
 	if failoverEndTime != emptyFailoverEndTime {
-		responseFailoverEndTime = &failoverEndTime
+		domainFailoverEndTime := failoverEndTime
+		responseFailoverEndTime = &domainFailoverEndTime
```

Comment:
nit: create a `TimePtr()` util func in `common/convert.go`?

Reply:
I replaced it with common.Int64Ptr